samples/nfc/system_off/src/main.c

```diff
@@ -138,12 +138,26 @@ static void system_off(struct k_work *work)
 
 	dk_set_led_off(SYSTEM_ON_LED);
 
-	int rc = pm_device_action_run(cons, PM_DEVICE_ACTION_SUSPEND);
+	if (IS_ENABLED(CONFIG_PM_DEVICE) && IS_ENABLED(CONFIG_SERIAL)) {
+		static const struct device *dev = DEVICE_DT_GET(DT_CHOSEN(zephyr_console));
+		int err;
+		enum pm_device_state state;
 
-	if (rc < 0) {
-		printk("Could not suspend console (%d)\n", rc);
+		if (dev) {
+			do {
+				err = pm_device_state_get(dev, &state);
+			} while ((err == 0) && (state == PM_DEVICE_STATE_ACTIVE));
+		}
 	}
 
+#ifdef CONFIG_SOC_NRF54L15_CPUAPP
+	/* Disable RAM retention in System OFF as it is not utilized by this sample. */
+	uint32_t ram_sections = 8;
+
+	nrf_memconf_ramblock_ret_mask_enable_set(NRF_MEMCONF, 0, BIT_MASK(ram_sections), false);
```

Comment:
Does disabling RAM retention have a big impact? It's SRAM so it shouldn't take much current to maintain it - on nrf54h20 SRAM retention uses up to 0.5uA - so not that much. Also the question is how much does the whole sample consume because if it matters is of course relative.